migrations/sqlite_versions/2021-02-10_a382a0f4b1be_correct_privacy_id_type_of_hostedby.py

```diff
@@ -0,0 +1,43 @@
+"""Correct privacy_id type of HostedBy
+
+Revision ID: a382a0f4b1be
+Revises: cb9a5f93f0aa
+Create Date: 2021-02-10 16:20:41.425779+00:00
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+import pepys_import
+
+# revision identifiers, used by Alembic.
+revision = "a382a0f4b1be"
+down_revision = "cb9a5f93f0aa"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
Forgive my being, dim, but. Should the above be "please do not adjust"?

Reply:
No, Alembic says that we should check the autogenerated migration scripts. That's why it is written: "please adjust!". Let me remove it.